plugins/techdocs-backend/src/service/router.ts

```diff
@@ -62,7 +62,13 @@ export async function createRouter({
   const router = Router();
 
   router.get('/metadata/mkdocs/*', async (req, res) => {
-    const storageUrl = config.getString('techdocs.storageUrl');
+    let storageUrl = config.getString('techdocs.storageUrl');
+    if (publisher instanceof LocalPublish) {
```

Comment:
I'm open to better solutions than this. This assumes that since the docs are published locally that we can access them locally on the same path as specified in the storageUrl and that's not necessarily true.

Reply:
Assume this would be alright for now until we have other publish methods @spotify/techdocs-core? Without having actually dug into this at all I'm assuming that we want to make this part of the publish API itself